src/catalog.ts

```diff
@@ -88,6 +91,7 @@ export class Dataset {
   constructor(
     readonly iri: IRI,
     readonly name: string,
+    readonly urlPrefix: IRI,
```

Comment:
Is this a good name?

Reply:
Renamed to `termsPrefixes`, as the semantics `IRI[]` is already clear from the type annotation.